ui.apps/src/main/content/jcr_root/apps/etoolbox-query-kit/console/query/clientlibs/js/actions/save.js

```diff
@@ -22,7 +22,9 @@
     function saveLatestQueries() {
         const query = ns.getEditorValue();
         const latestQueries = ns.DataStore.getLatestQueries();
-
+        if (!latestQueries || latestQueries[0] === query) {
```

Comment:
Hi!) I'd recommended that:
- you check not the very last `latestQuery` but all of them (imagine a person who has two different queries in clipboard and sends them 1, then 2, then 1, then 2...)
- if an equivalent query is found anywhere on the list, pop it to the top of the list (so that it becomes "the most latest")

Reply:
Fixed, thanks for meaningful remarks